docs/ref/templates/builtins.txt

```diff
@@ -2850,8 +2850,8 @@ If ``value`` is ``"https://www.example.org/"``, the output will be
 Converts URLs and email addresses in text into clickable links.
 
 This template tag works on links prefixed with ``http://``, ``https://``, or
-``www.``. For example, ``https://goo.gl/aia1t`` will get converted but
-``goo.gl/aia1t`` won't.
+``www.``. For example, ``https://djangoproject.com`` will get converted but
+``djangoproject.com`` won't.
```

Comment:
This example doesn't work because it has ".com" (see paragraph below). 
We can update to something like `https://google.de` and `google.de` (or something else if we don't want to mention google) 🤔 any preference?

Reply:
Sorry, I should check it more carefully. Maybe `https://djangocon.eu` :eu:

Reply:
Yes, sorry for not spotting that! Sounds good to me :)

Reply:
I like djangocon.eu 😁